app/controllers/games_controller.rb

```diff
@@ -13,17 +21,17 @@ def index
 
   # GET /games/1
   def show
-    render json: @game
+    render_game_with_state_and_player(@game)
```

Comment:
```suggestion
    render json: fn()
```
Let's leave the render call in the controller and only move the logic to create the json else where

Reply:
👍🏽